bot.py

```diff
@@ -291,6 +292,57 @@ async def watchlist_clear(interaction: nextcord.Interaction, watchlist_name):
     await interaction.response.send_message(response)
 
 
+@bot.slash_command(guild_ids=[GUILD_ID], name="add_tags", description="Add tags to a media in watchlist", default_member_permissions=EDIT_PERMISSION)
```

Comment:
I feel like tags is a better slash command name? Just cause if like you started typing add it would show up with the other add command where as starting to type tag would instantly only have this option? Not sure though - I don't mind it being add_tags.. up to you :))
```suggestion
@bot.slash_command(guild_ids=[GUILD_ID], name="tags", description="Add tags to a media in watchlist", default_member_permissions=EDIT_PERMISSION)
```

Reply:
I agree, however the next feature is going to be to allow users to remove tags, so I thought that it might be better to add some further clarity, I initially did have just 'tags' as you mentioned though.